app/src/swig/app.i

```diff
@@ -167,6 +168,49 @@ static void LogHeartbeatForDesktop(App* app) {
 #endif  // FIREBASE_PLATFORM_DESKTOP
 }
 
+static void RegisterLibraryHelper(const char* library, const char* version) {
```

Comment:
Can we delete RegisterLibraryInternal and RegisterLibraryHelper?
It looks like there is no current usage of RegisterLibraryInternal.

Alternatively, if RegisterLibraryInternal is still useful, can it be changed to just create a map with a single entry and then call RegisterLibrariesInternal?

That way we would not need to have these two very similar helpers.

Reply:
Good call. Deleted.